metrics/src/main/java/mobile/aerogear/org/metrics/MetricsConfig.java

```diff
@@ -0,0 +1,19 @@
+package mobile.aerogear.org.metrics;
+
+import org.aerogear.mobile.core.configuration.ServiceConfiguration;
+
+public class MetricsConfig {
```

Comment:
Why we need this if ServiceConfiguration already has the URI?

Reply:
+1 for having that as it may include security tokens and other stuff in near future. 
It's simple at the moment. I have the same on IOS

Reply:
So, do it in the near future, not now :)

We should not include stuff we don't need now IMO